.github/ISSUE_TEMPLATE/1.bug.yml

```diff
@@ -34,11 +34,11 @@ body:
     attributes:
       label: Steps to reproduce 🕹
       description: |
-        Provide a link to a live example (you can use codesandbox.io) and an unambiguous set of steps to reproduce this bug.
-        Include code to reproduce, if relevant (which it most likely is).
-
-        You can use the official codesandbox template as a starting point: https://mui.com/r/issue-template.
+        Please provide a link to a live example and an unambiguous set of steps to reproduce this bug.
+             
+        As a starting point, we recommend you browse our [documentation](https://mui.com/material-ui/getting-started/installation/), and [select](https://mui.com/static/core/forking-an-example.png) the closest example to your use case.
 
+        Or you can use the [official template](https://mui.com/r/issue-template) to build a reproduction case.
         If you have an issue concerning TypeScript please start from this TypeScript playground: https://mui.com/r/ts-issue-template.
```

Comment:
Is this still needed, considering the issue template mentioned above uses TypeScript?